indico/modules/events/templates/display/conference.html

```diff
@@ -0,0 +1,107 @@
+{% from 'attachments/_attachments.html' import render_attachments %}
+
+<div class="conferenceDetails">
+    <div itemprop="description" class="description">
+        {{ event.description }}
+    </div>
+
+    <div class="infogrid">
+        <div class="infoline date">
+            <span title="{% trans %}Date/Time{% endtrans %}" class="icon icon-time" aria-hidden="true"></span>
+            <div class="text">
+                <div>
+                    <span class="label">{% trans %}Starts{% endtrans %}</span>
+                    <time datetime="{{ event.start_dt_display.isoformat() }}">
+                        {{ event.start_dt_display|format_datetime(timezone=event.display_tzinfo) }}
+                    </time>
+                </div>
+                <div>
+                    <span class="label">{% trans %}Ends{% endtrans %}</span>
+                    <time datetime="{{ event.end_dt_display.isoformat() }}">
+                        {{ event.end_dt_display|format_datetime(timezone=event.display_tzinfo) }}
+                    </time>
+                </div>
+                <div class="timezone">{{ event.display_tzinfo.zone }}</div>
+            </div>
+        </div>
+
+        {% if event.has_location_info or event.address %}
+            <div class="infoline location">
+                <span title="{% trans %}Location{% endtrans %}" class="icon icon-location" aria-hidden="true"></span>
+                <div class="place text">
+                    {{ event.venue_name }}
+                </div>
+                {% if event.room and event.room.map_url %}
+                    <div class="room text">
+                        <a href="{{ event.room.map_url }}">{{ event.room_name }}</a>
+                    </div>
+                {% elif event.room_name %}
+                    <div class="room text">
+                        {{ event.room_name }}
+                    </div>
+                {% endif %}
+
+                {% if event.address %}
+                    <div class="address text nohtml">{{ event.address }}</div>
+                {% endif %}
+            </div>
+        {% endif %}
+
+        {% if event.person_links %}
+            <div class="infoline chairs clear">
+                <span title="{% trans %}Chairpersons{% endtrans %}" class="icon icon-user-chairperson"
+                      aria-hidden="true"></span>
+                <ul class="chair_list text">
```

Comment:
is `chair_list` a legacy class?

Reply:
yes, and some custom event stylesheets actually use it so i didn't want to change it